docs/input.md

```diff
@@ -102,6 +102,28 @@ print(result.output)
 #> The document discusses...
 ```
 
+## Uploaded files
+
+Use [`UploadedFile`][pydantic_ai.UploadedFile] when you've already uploaded content to the model provider.
```

Comment:
Related to the above, let's include examples of how to do that for all providers